content/sensu-core/0.29/guides/snmp-sensu-guide.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: "How to Monitor Devices With Sensu's SNMP Receiver Extension"
```

Comment:
I don’t necessarily think changing this is wrong, but I’ve noticed that the guides in our 2.0 documentation use a naming convention which tends to describe the problem space “how to do X with Y” instead of the solution space, e.g. “feature Z guide”. 

With this in mind, should we keep or revise the current title instead? I realize it’s a bit unwieldy currently.

Reply:
My motivation here was to use a shorter title that would be easier to see in the sidebar, but I just saw that we can use `linkTitle` for a shorter title in the menu bar! I'll fix this up. Thanks for the comment!